tests/unit/peapods/zmq/test_zmq_addr.py

```diff
@@ -10,4 +11,6 @@ def test_get_ctrl_addr(host):
 
 @pytest.mark.parametrize('host', ['pi@192.0.0.1', '192.0.0.1'])
 def test_get_ctrl_addr_ipc(host):
-    assert Zmqlet.get_ctrl_address(host, 56789, True)[0].startswith('ipc')
+    assert (
```

Comment:
Does this new assertion make sense? U mean, if `__windows__` this test should return another thing? isn't it better to skip it under `__windows__`?

Reply:
There's a separate logic in `get_ctrl_address` for windows already. 
https://github.com/jina-ai/jina/blob/d80c8d1aeb183bb8bbb552757fe937bafc770c66/jina/peapods/zmq/__init__.py#L133